paddle/trainer/tests/simple_sparse_neural_network.py

```diff
@@ -0,0 +1,23 @@
+from paddle.trainer_config_helpers import *
+
+settings(batch_size=128, learning_method=AdaGradOptimizer(), learning_rate=1e-4)
```

Comment:
in method
```cpp
void trainerOnePassTest(const string& configFile,
                         bool useGpu,
                         bool parallel,
                         int trainerCount = 1,
                         double averageWindow = 0.0f,
                         bool doAverageInCpu = false) ;
```

So i invoke trainerOnePassTest as

```cpp
trainerOnePassTest(configFileSimpleSparse, false, false, 1, 0.5, true);
```

Just set AverageWindow=0.5 in cpp side.